cuenca/resources/api_keys.py

```diff
@@ -16,6 +16,7 @@ class ApiKey(Creatable, Queryable, Retrievable, Updateable):
     secret: str
     deactivated_at: Optional[dt.datetime]
     user_id: Optional[str]
+    metadata: Optional[dict] = None
```

Comment:
why is this optional?

Reply:
it's because `cuenca.ApiKey.create()` doesn't require any parameters. Also, current core logic, It is not required to save `metadata` when create` ApiKey`.